app/models/repository_row.rb

```diff
@@ -84,6 +85,31 @@ class RepositoryRow < ApplicationRecord
   scope :active, -> { where(archived: false) }
   scope :archived, -> { where(archived: true) }
 
+  scope :in_repository_with_active_reminders, lambda { |repository|
```

Comment:
Can we just simplify it to a predefined repository?
`@repository.repository_rows.with_active_reminders`

Because I don't get point why we need this query for `repository_cell_ids` if we already will have scoped rows.

And how you will solve duplicate rows? If we have 2 reminders in one row then it will return duplicate rows